slownik.rst

```diff
@@ -0,0 +1,178 @@
+
+MVP
+  ang. Minimum Viable Product - podstawowa wersja systemu zapewniająca podstawową funkcjonalność umożliwiającą wprowadzenie na rynek.
+
+Marketplace
+  Marketplace będzie miejscem, gdzie przedstawiciel instytucji może zarejestrować się i utworzyć swoją instancję platformy aplikacje.gov.pl. Ten komponent będzie też odpowiadał za instalowanie poszczególnych aplikacji oraz komunikację z chmurą obliczeniową w celu np. skonfigurowania nowej instancji lub zmiany parametrów istniejącej instancji.
+
+Metadane
+  Zestaw usystematyzowanych informacji, logicznie powiązanych z przesyłką, sprawą lub inną dokumentacją, ułatwiających ich wyszukiwanie, kontrolę, zrozumienie i długotrwałe przechowanie oraz zarządzanie
+
+NASK
+  NASK jest instytutem badawczym prowadzącym działalność badawczo-rozwojową w zakresie opracowywania rozwiązań zwiększających efektywność, niezawodność i bezpieczeństwo sieci teleinformatycznych oraz innych złożonych systemów sieciowych oraz działalność edukacyjną, w tym w zakresie popularyzacji idei społeczeństwa informacyjnego
+
+Kod
+  Treść programu komputerowego.
+
+Komponent
+  Część składowa systemu realizująca konkretny zbiór funkcji.
+
+KPI (kluczowy wskaźnik efektywności)
+  Finansowy i niefinansowy wskaźnik pomiaru stopnia realizacji celów. Powinien być wyrażony w liczbach, procentach itd.
+
+Keyless Signatures’ Infrastructure
+  Usługa znakowania czasem w technologii linked timestamping, działająca zgodnie z modelem przedstawionym w dokumencie `„Keyless Signatures’ Infrastructure: How to Build Global Distributed Hash-Trees”`_.
+
+Operator techniczny
+  Podmiot, do którego Beneficjent – NASK przekaże pewne aspekty zarządu technicznego Platformy.
+
+PIUiD
+  Platforma Integracyjna Usług i Danych.
+
+Pieczęć elektroniczna
+  Dane w postaci elektronicznej dodane do innych danych w postaci elektronicznej lub logicznie z nimi powiązane, aby zapewnić autentyczność pochodzenia oraz integralność powiązanych danych.
+
+Plik JAR
+  (Tylko w kontekście JVM) najmniejsza możliwa do załadowania do środowiska uruchomieniowego część kodu.
+
+Podpis elektroniczny
+  Dane w postaci elektronicznej, które są dołączone lub logicznie powiązane z innymi danymi w postaci elektronicznej, i które użyte są przez podpisującego jako podpis.
+
+Poświadczenie odbioru
+  Urzędowe poświadczenie odbioru w rozumieniu ustawy z dnia 17 lutego 2005 r. o informatyzacji działalności podmiotów realizujących zadania publiczne.
+
+Podmioty komercyjne
+  Firmy świadczące usługi IT, która będą zamieszczać aplikacje swojej produkcji na Platformie.
+
+Portale usługowe
+  Witryny internetowe umożliwiające obywatelom dostęp do danych i funkcjonalności oferowanych przez systemy teleinformatyczne administracji publicznej.
+
+Profil Zaufany
+  Metoda potwierdzania tożsamości obywatela Polski w elektronicznych systemach administracji.
+
+Projekt / Platforma
+  aplkacje.gov.pl (EZD RP).
+
+Platforma
```

Comment:
Niefortunna nazwa, bo mamy też "instancję", która bywa nazywana (także w tym dokumencie) "instancją platformy". Jeśli zastosować definicję pod którą jest ten komentarz, to "instancja platformy" jest czymś innym niż "instancja platformy" z poprzedniego zdania.